xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -103,10 +103,10 @@
         <Parameter Name="leaveOpen" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <param name="mode">To be added.</param>
+        <param name="stream">The stream to compress.</param>
+        <param name="mode">One of the enumeration values that indicates whether to compress or decompress the stream.</param>
         <param name="leaveOpen">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Initializes a new instance of the <see. cref="T:System.IO.Compression.Brotli.BrotliStream" /> class by using the specified stream and compression mode, and optionally leaves the stream open.</summary>
```

Comment:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.IO.Compression.Brotli.BrotliStream" /> class by using the specified stream and compression mode, and optionally leaves the stream open.</summary>
```

Reply:
Thank you for catching those extra dots, I totally missed them.